experimental/builder/test/conv/test_ckb_conv_fwd_2d_bf16.cpp

```diff
@@ -20,10 +20,10 @@ TEST(FwdConvInstances,
     constexpr ThreadBlock FwdThreadBlock{.block_size = 256,
                                          .tile_size  = {.m = 256, .n = 256, .k = 32}};
 
-    run_test<FwdConvSignature,
-             FwdThreadBlock,
-             BlockGemmPipelineVersion::V1,
-             ConvFwdSpecialization::DEFAULT>();
+    run_test_DeviceGroupedConvFwdMultipleABD_Xdl_CShuffle_V3<FwdConvSignature,
```

Comment:
We're getting away from the builder concept here. Why should test code need to know about these old CK instances? Does this all break if we move to CK-tile? I know we need some internal translation in the library, but by the test layer this implemenation detail shouldn't be here.